eox_nelp/locale/config.yaml

```diff
@@ -0,0 +1,13 @@
+# Configuration for i18n workflow.
+
+locales:
+  - en # English - Source Language
+  - ar # Arabic
+
+# The locales used for fake-accented English, for testing.
+dummy_locales:
```

Comment:
is this still necessary ?

Reply:
no: [b4b0f9c](https://github.com/eduNEXT/eox-nelp/pull/19/commits/b4b0f9c296b1d214c1890a04a7c2f85c606982d0)